EntAppProject/src/main/resources/templates/JBUserProfile.html

```diff
@@ -2,36 +2,24 @@
 
 <html>
 
-<head>
-

```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.